### PR TITLE
Update release notes for 1.6.0.

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: doc/source/api_changes.rst [hunk 1]
```diff
@@ -93,13 +95,42 @@ now expect that the iterable passed in is callables with the signature::
   def fut_fac() -> awaitable:
       'This must work multiple times'
 
+The persistent dict used by ``RE.md`` must be thread-safe
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+By default, ``RE.md`` is an ordrinary dictionary, but any dict-like object may
```
Comment (jklynch) [inline doc/source/api_changes.rst]: "ordinary"

File: doc/source/api_changes.rst [hunk 2]
```diff
@@ -93,13 +95,42 @@ now expect that the iterable passed in is callables with the signature::
...
+* Added better logging and convenience functions for managing it more easily.
+  See :doc:`debugging`.
 * Generalized :func:`~bluesky.plans.list_scan` to work on any number of motors,
   not just one. In v1.2.0, :func:`~bluesky.plans.scan` was generalized in the
   same way.
 * Added :func:`~bluesky.plans.list_grid_scan`.
+* Added :func:`~bluesky.plan_stubs.rd`.
+* Added :class:`~bluesky.suspenders.SuspendWhenChanged`.
+* Added :func:`~bluesky.callbacks.core.make_callback_safe` and
+* Added an ``per_shot`` parameter to :func:`bluesky.plans.count`, analogous to
```
Comment (jklynch) [inline doc/source/api_changes.rst]: Added "a" ?

File: doc/source/callbacks.rst [hunk 3]
```diff
@@ -942,18 +942,29 @@ Ignoring Callback Exceptions
 ----------------------------
 
 If an exception is raised while processing a callback, the error can interrupt
-data collection. Usually, this is good: if, for example, the callback that is
-saving your data encounters an error, you want to know immediately.
+data collection. Sometimes, this is good: if, for example, the callback that is
+saving your data encounters an error, you want to know immediately rather than
+continuing to *think* you are collecting data when in fact it is being lost.
+But in many situations, such as visualization or first-pass data processing, it
+is usuallybetter for data collection to proceed even if a callback fails. These
```
Comment (jklynch) [inline doc/source/callbacks.rst]: "usually better"

File: doc/source/api_changes.rst [hunk 4]
```diff
@@ -93,13 +95,42 @@ now expect that the iterable passed in is callables with the signature::
...
+* Added support for :doc:`multi_run_plans`.
+* Added better logging and convenience functions for managing it more easily.
+  See :doc:`debugging`.
 * Generalized :func:`~bluesky.plans.list_scan` to work on any number of motors,
   not just one. In v1.2.0, :func:`~bluesky.plans.scan` was generalized in the
   same way.
 * Added :func:`~bluesky.plans.list_grid_scan`.
+* Added :func:`~bluesky.plan_stubs.rd`.
+* Added :class:`~bluesky.suspenders.SuspendWhenChanged`.
+* Added :func:`~bluesky.callbacks.core.make_callback_safe` and
```
Comment (mrakitin) [inline doc/source/api_changes.rst]: Incomplete sentence.

File: doc/source/api_changes.rst [hunk 5]
```diff
@@ -93,13 +95,42 @@ now expect that the iterable passed in is callables with the signature::
...
+* Added a ``per_shot`` parameter to :func:`bluesky.plans.count`, analogous to
+  the ``per_step`` parameter supported by plans that do scans.
+  :func:`~bluesky.callbacks.core.make_class_safe`.
```
Comment (mrakitin) [inline doc/source/api_changes.rst]: This looks to be a part of the incomplete sentence above.

**Pull request merged**